layers/meta-balena-imx8mplus/recipes-bsp/u-boot/patches/0003-integrate-with-balenaOS.patch

```diff
@@ -51,28 +52,39 @@ index ba8e706f8d1c..e055c64be0bd 100644
  	"mmcroot=" CONFIG_MMCROOT " rootwait rw\0" \
  	"mmcautodetect=yes\0" \
 -	"mmcargs=setenv bootargs ${jh_clk} console=${console} root=${mmcroot}\0 " \
-+	"mmcargs=setenv bootargs ${jh_clk} console=${console} ${resin_kernel_root} ${os_cmdline}\0 " \
++	"mmcargs=setenv bootargs ${jh_clk} earlycon=ec_imx6q,0x30890000,115200 console=${console} ${resin_kernel_root} ${root_opt} ${os_cmdline} "__stringify(BALENA_STAGE2)" \0 " \
```

Comment:
Indeed @mtoman I forgot to review this post-development. I  have removed setting the earlycon and console command line argument from here so that meta-balena can actually configure the production vs OS development consoles accordingly. I [PRed](https://github.com/balena-os/meta-balena/pull/3557) meta-balena to allow to set an earlycon too.